src/GameDataReader/Common/Refractor/V1/Files/GlobalRefractorV1ConfigFile.cs

```diff
@@ -5,17 +5,22 @@ namespace GameDataReader.Common.Refractor.V1.Files;
 internal class GlobalRefractorV1ConfigFile : RefractorConfigFile<GlobalRefractorV1ConfigFile>
 {
     private readonly string _modName;
+    private readonly string _gameProcessPath;
+    private readonly string _conFilePath;
 
     public GlobalRefractorV1ConfigFile(string gameName, string modName)
         : base(gameName)
     {
+        var utilityMethodsRefractorV1 = new UtilityMethodsRefractorV1ConfigFile();
         _modName = modName;
+        _gameProcessPath = utilityMethodsRefractorV1.GetProcessPath(_modName);
+        var conFile = $@"\Mods\{_modName}\Settings\Profile.con";
+        _conFilePath = utilityMethodsRefractorV1.GetConFilePath(conFile, _gameProcessPath, GameName);
```

Comment:
Since this is building the path to the config file, this logic would be better placed in `GetFilePath`. I don't see any immediate benefit of having it in the constructor.